reference_tag: 0
tags:
  0:
  - 0.0
  - 0.0
  - 0.0
  1:
  - -0.412567
  - -0.002883
  - 0.002174
  2:
  - -0.206142
  - -0.414625
  - 0.000375
  3:
  - 0.000901
  - -0.82863
  - 0.001302
  4:
  - -0.411886
  - -0.82947
  - -0.001407
  5:
  - -0.205423
  - -1.24269
  - 0.001081
  6:
  - 0.002013
  - -1.658149
  - 0.003144
  7:
  - -0.411246
  - -1.658701
  - 0.00178
  8:
  - -0.20341
  - -2.072051
  - 0.000589
  9:
  - 0.003824
  - -2.48557
  - 0.004697
  10:
  - -0.409146
  - -2.485291
  - -0.002421
  11:
  - -0.201221
  - -2.9002
  - 0.005099
  12:
  - 0.007456
  - -3.313567
  - 0.006582
  13:
  - -0.40668
  - -3.314207
  - 0.002242
  14:
  - -0.197741
  - -3.728308
  - 0.00582
  15:
  - 0.012621
  - -4.141515
  - 0.005872
  16:
  - -0.400768
  - -4.143603
  - 0.005055
  17:
  - -0.192228
  - -4.5559
  - 0.005977
  18:
  - 0.018257
  - -4.969545
  - 0.006354
  19:
  - -0.395462
  - -4.971127
  - 0.004048
  20:
  - -0.186902
  - -5.383662
  - 0.004885
  21:
  - 0.022296
  - -5.796802
  - 0.00611
  22:
  - -0.391515
  - -5.800005
  - 0.009994
  23:
  - -0.183287
  - -6.211584
  - 0.004537
